xil_cloud_telematics/carma_docker.launch

```diff
@@ -56,9 +56,6 @@
   <!-- Launch the CARMA-CARLA simulation -->
   <arg name="simulation_mode" default="true"/>
 
-  <!-- Rosbag Flag -->
```

Comment:
This is replaced by the `use_rosbag` parameters above right? Is this just part of the ros2 changes  we missed in the XIL config?

Reply:
Yes, this is correct

Reply:
These parameters were updated in this commit https://github.com/usdot-fhwa-stol/carma-config/commit/f3c6dc1cbf3cb090be6f35751fc54467aeb6cee4 to use_ros1_rosbag and use_ros2_rosbag. Looks like they were just moved to VehicleConfigParams.yaml